packages/backend/src/server.ts

```diff
@@ -8,8 +8,10 @@ telemetry.setServiceType('main');
 
 const port = appConfig.port;
 
+const serverUrl = `Server is listening on https://localhost:${port}/`;
```

Comment:
Instead of hardcoding `localhost` here, it's beneficial to use the necessary values from `appConfig`. For example, we have `appConfig.apiUrl` that shows where the API can be accessed at. We may use it here.

```suggestion
const serverUrlInfoText = `Server is listening on ${appConfig.apiUrl}`;
```

I also suggest that we rename the variable according to what it stores. As it's storing more than `url`, `serverUrlInfoText` may suit.

p.s. Keep in mind that the logged address would be directly for the API. So, when the web app is separately served (out of the API), the logged address would take the user to the API instead of the web application. For now, this should be fine.